Project.toml

```diff
@@ -1,13 +1,13 @@
 name = "DataStructures"
 uuid = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-version = "0.14.2"
+version = "0.15.0"
 
 [deps]
 InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 
 [compat]
-OrderedCollections = "1.0.1"
+OrderedCollections = "1.1.0"
```

Comment:
should have `compat` for Julia?

Reply:
yes, and my 0.14.2 tag is about to be failed because of that.